float32[] position
float32[] velocity
float32[] attitude
float32 timestamp